File: level 18/homework/hw11.py
#  11) მომხმარბელს შემოატანინეთ რიცხვი შემდეგ კი მომხმარებლის შემოტანილ რიცხვამდე დაბეჭდეთ მხოლოდ კენტი რიცხვები

number=int(input('Enter number'))
for i in range(number):
    if i%2!=0:
        print(i)

File: level 115/homework/js/hw11.js
// 11)შექმენი პატარა “მათემატიკური თამაში”:
// დააგენერირე შემთხვევითი რიცხვი 1-დან 10-მდე.
// მომხმარებელს შემოატანინე რიცხვი.
// თუ მომხმარებლის რიცხვი დაემთხვა დაგენერირებულს — გამოიტანე "სწორია!",
// სხვა შემთხვევაში "ცდები!".

let random=Math.floor(Math.random()*10)

let userNum=Number(prompt('enter numberrr'))
if(userNum===random){
    console.log( "სწორია!")
}else{
    console.log("ცდები!")
}
console.log('random number:',random, 'your number:',userNum)